pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -141,6 +141,12 @@ func GetAPIClient() (*APIClient, error) {
 	return globalAPIClient, nil
 }
 
+// ReSetGlobalAPIClientOnce resets globalAPIClientOnce
+func ReSetGlobalAPIClientOnce() {
```

Comment:
"Reset" is one word.

```suggestion
func ResetGlobalAPIClientOnce() {
```